test/saithrift/tests/switch.py

```diff
@@ -51,6 +51,21 @@
 
 is_bmv2 = ('BMV2_TEST' in os.environ) and (int(os.environ['BMV2_TEST']) == 1)
 
+#COUNTERS
+COUNTERS_EGRESS_DROP = 0
+COUNTERS_INGRESS_DROP = 1
+COUNTERS_PFC_PRIO_0 = 2
+COUNTERS_PFC_PRIO_1 = 3
+COUNTERS_PFC_PRIO_2 = 4
+COUNTERS_PFC_PRIO_3 = 5
+COUNTERS_PFC_PRIO_4 = 6
+COUNTERS_PFC_PRIO_5 = 7
+COUNTERS_PFC_PRIO_6 = 8
+COUNTERS_PFC_PRIO_7 = 9
+COUNTERS_TRANSMITTED_OCTETS = 10
+COUNTERS_TRANSMITTED_PKTS = 11
+COUNTERS_RECEIVED_PKTS = 12
```

Comment:
I can delete it and it still gonna work
but if in the future someone will want to use the read counter function 
(sai_thrift_read_port_counters)
it's return an array, and the use of const will make it easy to understand the array fields

*when I run the test the first time 
I wanted to see in my eyes the actual numbers of the counters